<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="chrome=1">

    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-dark.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print">

    <title>Bioinformatics-scripts by lakhujanivijay</title>
  </head>

  <body>

    <header>
      <div class="container">
        <h1>Bioinformatics-scripts</h1>
        <h2>This repository consists of specific bioinformatics problems solved by writing small scripts; majority of them in Python. The problems have been taken from open platforms like ROSALIND (http://rosalind.info/problems). Some of the scripts are solutions to my day-to-day challenges while working.</h2>

        <section id="downloads">
          <a href="https://github.com/lakhujanivijay/Bioinformatics-Scripts/zipball/master" class="btn">Download as .zip</a>
          <a href="https://github.com/lakhujanivijay/Bioinformatics-Scripts/tarball/master" class="btn">Download as .tar.gz</a>
          <a href="https://github.com/lakhujanivijay/Bioinformatics-Scripts" class="btn btn-github"><span class="icon"></span>View on GitHub</a>
        </section>
      </div>
    </header>

    <div class="container">
      <section id="main_content">
        <h3>
<a id="about-me" class="anchor" href="#about-me" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>About Me</h3>

<p>❝ A biologist by heart, a bioinformatician by brain.❞</p>

<p>☛     Genomics Domain Specialization: NET-JRF, GATE, BINC qualified.</p>

<p>☛     Certified bioinformatician (by Dept. of Biotechnology, Govt. of India), All India Rank 18, BINC 2012 (<a href="http://www.pondiuni.edu.in/exams/binc/rs2012.html">http://www.pondiuni.edu.in/exams/binc/rs2012.html</a>).</p>

<p>☛      Masters in Bioinformatics with 4+ years of experience in microarray/library probe design,image analysis, data analysis for CGH, ChIP and CH3 arrays.</p>

<p>☛     2.5 years of experience of next generation sequencing data analysis software from one of the global leaders.</p>

<p>☛     Skilled in working with commercial/open source software for microarray and NGS end-to-end data analysis, from raw sequencing data to variant reporting. Whole genome,transcriptome, 16S rRNA NGS data analysis, de novo assembly tools &amp; algorithms, assembly validation and downstream data processing.</p>

<p>☛     Sound knowledge of open source tools for data visualization and interpretation. Comfortable with working on Linux platform &amp; HPC.</p>

<h3>
<a id="authors-and-contributors" class="anchor" href="#authors-and-contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Authors and Contributors</h3>

<p>Author:Vijay Lakhujani</p>

<p>Github: <a href="https://github.com/lakhujanivijay" class="user-mention">@lakhujanivijay</a></p>

<p>Linkedin: <a href="https://in.linkedin.com/in/lakhujanivijay">https://in.linkedin.com/in/lakhujanivijay</a></p>
      </section>
    </div>

    
  </body>
</html>
